### PR TITLE
add genesis launcher files

### PR DESCRIPTION
**Pull request by matteematt**
ATENTION! YOU SHOULD NOW BRANCH FROM PREPROD WHEN YOU UPDATE 

  - Please check the [internal contributions guide](https://www.notion.so/genesisglobal/Contributing-new-documentation-75953fb245f246ff872789035451a0c4) for details
  
__________

Thank you for contributing to the documentation.

Do the changes you have made apply to both Current and Previous versions?

Have you done a trial build to check all new or changed links?

Is there anything else you would like us to know?

**This week's exciting advice from the style guide**

- Write your headings in sentence case:

  - A good example
    This is a correct heading.
  - A Bad Example
    This is not a correct heading.

### CODE REVIEW

**Comment by jay-taylerson** [comment]
I personally think we should separate steps 1 + 2 from Step 3, and put in their own page...

- Add dependency to project page, which contains step 1 and 2, and on the main page above this add a message along the lines of 
- Change Step 3 title to "Run the launcher" 

Why? Because we should also add a "If you created your project on an initial version after X your project likely has the dependencies and you can skip this step. If you have trouble running it please check the setup"

I also think we should have a bit under "Run the Launcher" which is how to "change the version of the launcher" and guides to the 1 place you need to update, and where in artifactory you can find the latest version

File: docs/03_server/12_tooling/03_genesis-launcher/02_running.mdx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  - server
+  - tooling
+  - code snippets
+  - launcher
+---
+
+## Starting the Launcher
+
+After syncing your gradle project, there should now be a `genesisLauncher` gradle task.
+You can start the launcher by running that task from an IDE, or from the command line, run:
```
Comment (jay-taylerson) [inline docs/03_server/12_tooling/03_genesis-launcher/02_running.mdx:16]: We should state which directory of the project to be in (even if root)

Comment (jay-taylerson) [reply docs/03_server/12_tooling/03_genesis-launcher/02_running.mdx:16]: looks like you need to be in the server directory

File: docs/03_server/12_tooling/03_genesis-launcher/02_running.mdx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+Once you open Genesis Launcher for your project, all the services will be listed, including the Web UI.
+To get started for the first time; click on the 🚀icon, just underneath the Genesis logo.
+
+This will bootstrap the database and launch the application.
+Once the database has been initialised, feel free to run "Start All" instead.
+
+Bootstrap will run "Start All" as well, but will first run [remap](/operations/commands/remap/) and ConsolidateRights.
+
+Prior to starting service, build tasks will be run.
+
```
Comment (jay-taylerson) [inline docs/03_server/12_tooling/03_genesis-launcher/02_running.mdx:33]: We should add an explicit section around compact processes and enabling it to save local PC resource utilisation (as this comes up a lot with clients)

image

File: docs/03_server/12_tooling/03_genesis-launcher/01_install.mdx
Comment (wjhendry) [inline docs/03_server/12_tooling/03_genesis-launcher/01_install.mdx:1]: Is it the case that you only need to do this enabling manually if you have not started with Create?
If so, we should make it very clear that you can ignore these steps, unless you are trying to work on a non-Create project.

**Pull request merged**